tools/github-promote-release.ts

```diff
@@ -25,26 +25,28 @@ const getReleaseId = async ({ deliverinoPrivateKey, owner, repo, platform }: Opt
   const result = releases.data.find(release => release.tag_name.includes(platform))
   if (result && result.prerelease) {
     console.log('Unset prerelease tag of ', result.tag_name)
-    return result.id
+    return { id: result.id, body: result.body }
   }
 
   console.log('No release found to unset the prerelease tag for. Latest release may already be non-prerelease')
   return null
 }
 
 const removePreRelease = async ({ deliverinoPrivateKey, owner, repo, platform }: Options) => {
-  const releaseId = await getReleaseId({ deliverinoPrivateKey, owner, repo, platform })
-  if (releaseId !== null) {
+  const release = await getReleaseIdBody({ deliverinoPrivateKey, owner, repo, platform })
+  if (release?.id !== null) {
     const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
     const result = await appOctokit.rest.repos.updateRelease({
       owner,
       repo,
-      release_id: releaseId,
+      release_id: release?.id,
       prerelease: false,
       make_latest: platform === 'android' ? 'true' : 'false', // We always want android to be the latest release, so a link to the latest github release will go to the apk
     })
     console.log('Http response code of updating the result: ', result.status)
+    return `The most recent beta version was promoted to production:\n ${release?.body}`
```

Comment:
:wrench: Just return the whole release here and only construct the message below in the action :)